### DISCUSSION TITLE
curso 7195

### DISCUSSION THREAD

**Discussion by jramaral** [balta-io/forum#300] [Tira Dúvidas] [answered]
Olá, estava fazendo o curso 7195 flutter , perfect eggs, mas estou tendo dificuldades, a minha versão  sdk é a >=3.0.0-313.0 e a do curso é  ">=2.1.0 <3.0.0", sendo assim muita, partes do codigo estão dando erro para a versão que estou usando. Por exemplo,  no arquivo theme.dart a maioria das propriedades estão deprecados na versão 3...
 Neste caso devo instalar a versão do curso, ou tem alguma atualização do código para a versão mais atual?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @jramaral, como vai?
Realmente, muita coisa mudou, minha sugestão é usar as mesmas versões do curso. Depois que terminar todas as aulas, ai você atualiza e testa com as novas SDKs!

Assim você aproveita melhor o curso.

**Reply by jramaral** [reply]
Legal, vou fazer isso . O app do Alcool ou Gasolina eu fiz uma publicação web dele, depois dá uma olhadinha quando puder. https://eog-jdamaral.vercel.app/#/
Sigo nos estudos. Abs.
